app/views/forecast_snapshots/show.html.erb

```diff
@@ -229,7 +229,7 @@
           alignTicks: false
         },
         title: {
-          text: 1 + i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
+          text: i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
```

Comment:
OK, I see where I oversimplified this and missed the fact that it was still starting from 0, but.... you didn't mean to incr the series id, did you? New version coming soon...